mappings/net/minecraft/test/StartupParameter.mapping

```diff
@@ -1,4 +1,239 @@
+	METHOD method_35955 expectEntityAt (Lnet/minecraft/class_1297;Lnet/minecraft/class_2338;)V
+		ARG 1 entity
+		ARG 2 pos
+	METHOD method_35959 expectEntity (Lnet/minecraft/class_1299;)V
+		ARG 1 type
+	METHOD method_35960 expectEntityToTouch (Lnet/minecraft/class_1299;DDD)V
+		ARG 1 type
+		ARG 2 x
+		ARG 4 y
+		ARG 6 z
+	METHOD method_35961 spawnEntity (Lnet/minecraft/class_1299;FFF)Lnet/minecraft/class_1297;
+		ARG 1 type
+		ARG 2 x
+		ARG 3 y
+		ARG 4 z
+	METHOD method_35962 spawnEntity (Lnet/minecraft/class_1299;III)Lnet/minecraft/class_1297;
+		ARG 1 type
+		ARG 2 x
+		ARG 3 y
+		ARG 4 z
+	METHOD method_35963 spawnEntity (Lnet/minecraft/class_1299;Lnet/minecraft/class_243;)Lnet/minecraft/class_1297;
+		ARG 1 type
+		ARG 2 vec3d
+	METHOD method_35964 spawnEntity (Lnet/minecraft/class_1299;Lnet/minecraft/class_2338;)Lnet/minecraft/class_1297;
+		ARG 1 type
+		ARG 2 pos
+	METHOD method_35965 expectEntityAround (Lnet/minecraft/class_1299;Lnet/minecraft/class_2338;D)V
+		ARG 1 type
+		ARG 2 pos
+		ARG 3 radius
+	METHOD method_35966 (Lnet/minecraft/class_1309;)Lnet/minecraft/class_1309;
+		ARG 1 entity
+	METHOD method_35967 (Lnet/minecraft/class_1308;Lnet/minecraft/class_2338;F)Lnet/minecraft/class_4693;
+		ARG 1 entity
+		ARG 2 pos
+	METHOD method_35968 spawnItem (Lnet/minecraft/class_1792;FFF)Lnet/minecraft/class_1542;
+		ARG 1 item
+		ARG 2 x
+		ARG 3 y
+		ARG 4 z
+	METHOD method_35969 expectItemAt (Lnet/minecraft/class_1792;Lnet/minecraft/class_2338;D)V
+		ARG 1 item
+		ARG 2 pos
+		ARG 3 radius
+	METHOD method_35970 expectItemsAt (Lnet/minecraft/class_1792;Lnet/minecraft/class_2338;DI)V
+		ARG 1 item
+		ARG 2 pos
+		ARG 3 radius
+		ARG 5 amount
+	METHOD method_35971 expectBlock (Lnet/minecraft/class_2248;III)V
+		ARG 1 block
+		ARG 2 x
+		ARG 3 y
+		ARG 4 z
+	METHOD method_35972 expectBlock (Lnet/minecraft/class_2248;Lnet/minecraft/class_2338;)V
+		ARG 1 block
+		ARG 2 pos
+	METHOD method_35974 (Lnet/minecraft/class_2680;Lnet/minecraft/class_2248;Lnet/minecraft/class_2248;)Z
+		ARG 2 block1
+	METHOD method_35976 (Lnet/minecraft/class_2769;Ljava/lang/Comparable;Lnet/minecraft/class_2680;)Z
+		ARG 2 state
+	METHOD method_35977 expectBlocksWithStateOf (Lnet/minecraft/class_3341;Lnet/minecraft/class_2338;)V
+		ARG 1 chackedBlockBox
+		ARG 2 correctStatePos
+	METHOD method_35978 getAbsolute (Lnet/minecraft/class_243;)Lnet/minecraft/class_243;
+	METHOD method_35979 (Lnet/minecraft/class_243;Lnet/minecraft/class_1297;)Z
+		ARG 1 entity
+	METHOD method_35980 getBlockState (Lnet/minecraft/class_2338;)Lnet/minecraft/class_2680;
+		ARG 1 pos
+	METHOD method_35981 putAndRemoveRedstoneBlock (Lnet/minecraft/class_2338;J)V
+		ARG 1 pos
+		ARG 2 delay
+	METHOD method_35982 expectEntityWithData (Lnet/minecraft/class_2338;Lnet/minecraft/class_1299;Ljava/util/function/Function;Ljava/lang/Object;)V
+		ARG 1 pos
+		ARG 2 type
+		ARG 3 entityDatagGetter
+		ARG 4 data
+	METHOD method_35983 expectContainerWith (Lnet/minecraft/class_2338;Lnet/minecraft/class_1792;)V
```

Comment:
Should we use `ScreenHandler` here?

Reply:
I think Container is better, because a ScreenHandler doesn't need to represent a container.

Reply:
Consistency with other yarn names is important. Though string constants may say `container` it needs to match the term everyone else.

Reply:
This isn't related to a screen handler, though. The parameters are `BlockPos` and `Item`, so it seems that this really expects a *container block* at the position containing an item. (Haven't checked the code for this, but these parameters have nothing to do with SHs)

Reply:
Yeah, my guess was correct. Both this method and `expectEmptyContainer` check for a `LockableContainerBlockEntity` at the position with the specific item requirements (contains item X or empty, respectively). Naming these using `ScreenHandler` would be wrong.

Side note: they throw `TimeMismatchException`s if the assertion fails but that exception name seems wrong... Are they more like generic test exceptions?

Reply:
Ah good, then we call it containers.